backend/domain/DTO/response/ListVoteDTO.js

```diff
@@ -1,8 +1,11 @@
+const evotes = require('../../../../common/Enum/Vote');
```

Comment:
why is the name here 'evotes' but in other files, it is 'votes'. Also, a more descriptive name for this variable might be useful. What is it representing? 'votes' is semi ambiguous.

Reply:
I agree. Perhaps VoteType?